docs/API-functions.md

```diff
@@ -1278,6 +1301,105 @@ of error has occurred on the socket.
 This is reported only if `msTimeOut` was \>=0, otherwise the function waits
 indefinitely.
 
+### srt_epoll_uwait
+```
+int srt_epoll_uwait(int eid, SRT_EPOLL_EVENT* fdsSet, int fdsSize, int64_t msTimeOut);
+```
+
+This function blocks a call until any readiness state occurs in the epoll
+container. Unlike `srt_epoll_wait`, it can only be used with `eid` subscribed
+to user sockets (SRT sockets), not system sockets.
+
+This function blocks until the timeout. If timeout is 0, it exits 
+immediately after checking. If timeout is -1, it blocks indefinitely 
+until a readiness state occurs.
+
+* `eid`: epoll container
+* `fdsSet` : A pointer to an array of `SRT_EPOLL_EVENT`
+* `fdsSize` : The size of the fdsSet array
+* `msTimeOut` : Timeout specified in milliseconds, or special values (0 or -1):
+   * 0: Don't wait, return immediately (report any sockets currently ready)
+   * -1: Wait indefinitely.
+
+- Returns:
+
+  * The total amount of user socket (SRT socket) states changed. Note that this
+can be larger than `fdsSize`, in this case and for events with `SRT_EPOLL_ET` flag
+only `fdsSize` events are returned in the `fdsSet` and the remaining events can
+be retrieved by a new call to this function
+
+  * 0, if no sockets were ready up to the timeout (never if waiting indefinitely)
+
+  * -1 in case of error
+
+
+- Errors:
+
+  * `SRT_EINVPOLLID`: `eid` designates no valid EID object
```

Comment:
This description is same in all other functions and it was approved previously by Steve. This can still be changed, but better one description be consistent for every function.

Reply:
It is not the same in all other functions. For example, just below it is "eid parameter doesn't refer to a valid epoll container" and that is much better.

Reply:
Well, if so then all should be made the same then.